specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkManagerActiveConfiguration.json

```diff
@@ -0,0 +1,240 @@
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConnectivityConfigurationsGet": {
+            "$ref": "./examples/NetworkManagerActiveConfigurationGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ActiveConfiguration": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ActiveConfigurationProperties",
+          "description": "The Active Configuration properties"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Configuration and location lists to commit."
+    },
+    "ActiveConfigurationProperties": {
+      "properties": {
+        "commits": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommitItem"
+          },
+          "description": "configuration Id need to deploy."
+        },
+        "deploymentTime": {
+          "type": "string",
+          "description": "Deployment time string",
+          "format": "date-time"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the peering Configuration resource."
+        }
+      },
+      "description": "Configuration and location lists to deploy."
+    },
+    "CommitItem": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CommitsProperties",
+          "description": "The Commit properties"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
```

Comment:
the api just returns existing resources and there is no resource such as 'commitItem'. so remove concept of 'commit'  in this API and project security/connectivity config directly under properties.

Reply:
We are not returning just security connectivity config. For security, it is a resource combined with configuration, rules, detailed groups, and properties such as deployment time.